CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 1.1.9
+  - Fix `@metadata` get overwritten [#34](https://github.com/logstash-plugins/logstash-input-dead_letter_queue/pull/34)
```

Comment:
I would only rephrase it because reading it's not so immediately to understand what it does.
Something like "Reestablish the DLQ stored event's metadata, avoiding to overwrite with newly created" or similar